pkg/roachprod/roachprod.go

```diff
@@ -769,8 +769,7 @@ func Start(
 	if err = c.Start(ctx, l, startOpts); err != nil {
 		return err
 	}
-	updatePrometheusTargets(ctx, l, c)
-	return nil
+	return UpdateTargets(ctx, l, clusterName, clusterSettingsOpts...)
```

Comment:
Why is this change necessary?

Reply:
Nm, I see that's because of the stripping of the node suffix from the clusterName. In that case, we should update the comment for `updatePrometheusTargets` that it _requires_ cluster names without node selector; otherwise, the same bug could resurface later if someone mistakenly uses `updatePrometheusTargets` instead of ` UpdateTargets`.

Reply:
This makes sense, I will actually move the split code to the updatePrometheusTargets.